src/Grpc.StatusProto/README.md

```diff
@@ -0,0 +1,256 @@
+﻿# Grpc C# API for error handling with Status.proto
+
+This is a protoype NuGet package providing client and server side support for the
+[gRPC richer error model](https://grpc.io/docs/guides/error/#richer-error-model).
+
+It had dependencies NuGet packages on:
+* `Google.Api.CommonProtos` - to provide the proto implementations used by the richer error model
+* `Grpc.Core.Api` - for API classes such as `RpcExcetion`
+
+Reviewers notes:
+
+* The client side borrows from ideas used in [googleapi/gax-dotnet](https://github.com/googleapis/gax-dotnet), specifically [RpcExceptionExtensions.cs](https://github.com/googleapis/gax-dotnet/blob/main/Google.Api.Gax.Grpc/RpcExceptionExtensions.cs)
```

Comment:
so the plan is that RpcExceptionExtensions from this PR will be added to Google.Api.CommonProtos and after those changes are released, Google.Api.Gax.Grpc will migrate to RpcExceptionExtensions from Google.Api.CommonProtos?

Reply:
Yes for the first part. For the second part, we'd probably migrate the implementation immediately (making the existing extension methods call the new ones) but we can't remove them from the code itself until the next major version of GAX.